diagnostics/plot.C

```diff
@@ -214,7 +222,8 @@ void plot(){
     
     h_ReprojectionError[iimg]->SetLineWidth(3);
     if (!iimg) h_ReprojectionError[iimg]->SetLineWidth(5);
-    h_ReprojectionError[iimg]->SetLineColor(iimg-1 + (!iimg)*2);
+    //h_ReprojectionError[iimg]->SetLineColor(iimg-1 + (!iimg)*2);
+    h_ReprojectionError[iimg]->SetLineColor(iimg+1);
```

Comment:
What did (!iimg)*2 do before?

Reply:
just another way of counting to set different colors, i forget exactly now